src/Esy.re

```diff
@@ -127,7 +127,9 @@ let processDeps = (dependenciesJson, folder) => {
         | Some(bsPlatformVersionJson) =>
           switch (Js.Json.classify(bsPlatformVersionJson)) {
           | JSONString(bsPlatformVersion) =>
-            if (bsPlatformVersion->Semver.minVersion->Semver.satisfies(">=6.0.0")) {
+            if (bsPlatformVersion
+                ->Semver.minVersion
+                ->Semver.satisfies(">=6.0.0")) {
```

Comment:
Sorry for unrelated changes, previously this file wasn't formatted properly.
Everything in this file below this point is just formatting